packages/strapi-admin/admin/src/containers/AuthPage/components/Oops/index.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Button, Padded, Text } from '@buffetjs/core';
+import { useHistory } from 'react-router-dom';
+import { useIntl } from 'react-intl';
+import Logo from '../Logo';
+import Section from '../Section';
+
+const Oops = () => {
+  const { push } = useHistory();
+  const { formatMessage } = useIntl();
+  const handleClick = () => {
+    push('/auth/login');
+  };
+
+  // TODO add logo when available
+
+  return (
+    <>
+      <Section textAlign="center">
+        <Logo />
+      </Section>
+      <Section withBackground textAlign="center">
+        <Padded top size="168px">
```

Comment:
What padding 😆 . Is there any reason for this one?

Reply:
Yes until I get the export of the oops picture.

lol